modelscope/__init__.py

```diff
@@ -60,6 +62,8 @@
         ],
         'hub.api': ['HubApi'],
         'hub.snapshot_download': ['snapshot_download'],
+        'hub.dataset_download':
+        ['dataset_snapshot_download', 'dataset_file_download'],
```

Comment:
dataset_snapshot_download  -> dataset_download   may be better?

Reply:
dataset_file_download and dataset_snapshot_download are a pair .